test/e2e_federation/federated-service.go

```diff
@@ -144,6 +155,7 @@ var _ = framework.KubeDescribe("[Feature:Federation]", func() {
 				By(fmt.Sprintf("Verified that services were not deleted from underlying clusters"))
 			})
 		})
+
```

Comment:
But isn't skipping **Service creation** the right thing? "Service creation should be deleted" doesn't make any sense right?